perun-core/src/main/java/cz/metacentrum/perun/core/bl/GroupsManagerBl.java

```diff
@@ -1190,7 +1215,98 @@
 	 * @throws WrongAttributeAssignmentException
 	 * @throws WrongAttributeValueException
 	 */
-	void saveInformationAboutGroupSynchronization(PerunSession sess, Group group, boolean failedDueToException, String exceptionMessage) throws AttributeNotExistsException, InternalErrorException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException, WrongAttributeValueException;
+	void saveInformationAboutGroupSynchronizationInNewTransaction(PerunSession sess, Group group, boolean failedDueToException, String exceptionMessage) throws AttributeNotExistsException, InternalErrorException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException, WrongAttributeValueException;
```

Comment:
For all these methods (with information about used transaction) should be better known how can be such method used correctly. For example "With new transaction, rollback on main method, where this one is used, will not revert saving of these information" or something similar (I am not sure if even I understand it right, so my version can be incorrect).